README.md

```diff
@@ -11,6 +11,18 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 
+
+## Install
+
+Manually install [Git Large File Storage](https://git-lfs.github.com/), then open a terminal and run:
+```
+git clone https://github.com/mozilla/DeepSpeech
+cd DeepSpeech
+git lfs fetch
```

Comment:
confirmed that the lfs commands are unnecessary if git-lfs  is installed before hand. Removed.